sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced-mlflow.ipynb

```diff
@@ -533,14 +614,501 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_file = \"forecast.json\"\n",
+    "batch_deployment = BatchDeployment(\n",
+    "    name=\"non-mlflow-deployment\",\n",
+    "    description=\"this is a sample non-mlflow deployment\",\n",
+    "    endpoint_name=batch_endpoint_name,\n",
+    "    model=registered_model,\n",
+    "    code_path=\"./forecast\",\n",
+    "    scoring_script=\"forecasting_script.py\",\n",
+    "    environment=env,\n",
+    "    environment_variables={\n",
+    "        \"TARGET_COLUMN_NAME\": target_column_name,\n",
+    "    },\n",
+    "    compute=cluster_name,\n",
+    "    instance_count=2,\n",
+    "    max_concurrency_per_instance=2,\n",
+    "    mini_batch_size=10,\n",
+    "    output_action=BatchDeploymentOutputAction.APPEND_ROW,\n",
+    "    output_file_name=output_file,\n",
+    "    retry_settings=BatchRetrySettings(max_retries=3, timeout=30),\n",
+    "    logging_level=\"info\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, start a model deployment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.begin_create_or_update(batch_deployment)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We need to create the Input, representing URI folder, because the batch endpoint is intended to process multiple files at a time. In this example we will use only one test file, which we have uploaded to the blob storage earlier. This file must be available through the url link."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create an inference job."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = ml_client.batch_endpoints.invoke(\n",
+    "    endpoint_name=batch_endpoint_name,\n",
+    "    input=my_test_data_input,\n",
+    "    deployment_name=\"non-mlflow-deployment\",  # name is required as default deployment is not set\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will stream the job output to monitor the execution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_name = job.name\n",
+    "batch_job = ml_client.jobs.get(name=job_name)\n",
+    "print(batch_job.status)\n",
+    "# stream the job logs\n",
+    "ml_client.jobs.stream(name=job_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download the prediction result for metrics calculation\n",
+    "The output of forecast output is saved in JSON format. You can download it and calculation some error metrics for the forecasts and vizualize the predictions vs. the actuals."
```

Comment:
Please replace "You can download it and calculation some error metrics for the forecasts You can download it and calculation some error metrics for the forecasts and vizualize the predictions vs. the actuals."" with

"You can use it to calculate test set metrics and plot predictions and actuals over time."

Reply:
Fixed